test/integration/MultichainProposal.t.sol

```diff
@@ -2621,4 +2621,95 @@ contract MultichainProposalTest is
         }
         require(userProposalFound, "proposal not created");
     }
+
+    function testGrantGuardianRoleAfterPause() public {
+        vm.selectFork(moonbeamForkId);
+
+        address pauseGuardian = addresses.getAddress(
+            "MOONBEAM_PAUSE_GUARDIAN_MULTISIG"
+        );
+
+        vm.prank(pauseGuardian);
+        governor.pause();
+
+        assertTrue(governor.paused(), "governor not paused");
+
+        vm.prank(pauseGuardian);
+        governor.unpause();
+
+        assertFalse(governor.paused(), "governor paused");
+        assertEq(governor.pauseGuardian(), address(0), "guardian not kicked");
+
+        /// mint whichever is greater, the proposal threshold or the quorum
+        uint256 mintAmount = governor.proposalThreshold() > governor.quorum()
+            ? governor.proposalThreshold()
+            : governor.quorum();
+
+        deal(address(well), address(this), mintAmount);
+        well.transfer(address(this), mintAmount);
```

Comment:
test passing without this transfer as well. All other tests in this file are also passing where this transfer is used without this loc.